go/pkg/services/logcollector/server.go

```diff
@@ -655,6 +665,9 @@ func (s *Server) startLogStreaming(ctx context.Context,
 		return
 	}
 
+	// add log file path to cache
+	s.logFilesCache.Set(s.getLogFilCacheKey(runUID, projectName), logFilePath, s.logFilesCacheTTL)
```

Comment:
```suggestion
	s.logFilesCache.Set(s.getLogFileCacheKey(runUID, projectName), logFilePath, s.logFilesCacheTTL)
```
change function name